core/src/verifier.rs

```diff
@@ -93,28 +93,22 @@ where
         // For now we multiply by 2 since we do not give signatures for burn_txs. // TODO: Change this in future.
         let num_required_nonces = 2 * self.operator_xonly_pks.len() + 1;
 
-        let mut dbtx = self.db.begin_transaction().await?;
         // Check if we already have pub_nonces for this deposit_outpoint.
-        let pub_nonces_from_db = self
-            .db
-            .get_pub_nonces(Some(&mut dbtx), deposit_outpoint)
-            .await?;
-        if let Some(pub_nonces) = pub_nonces_from_db {
+        if let Some(pub_nonces) = self.db.get_pub_nonces(None, deposit_outpoint).await? {
```

Comment:
Let's keep it as is, but can you create an issue @ceyhunsen ?

Reply:
reverted in b90bb81b2f85479271873a7d05af425dfbe925ff
issue: https://github.com/chainwayxyz/clementine/issues/325